dataminer/Administrator_guide/DataMiner_Systems/BPA_tests/BPA_NATS_Migration_Prerequisites.md

```diff
@@ -4,6 +4,7 @@ uid: BPA_NATS_Migration_Prerequisites
 
 # Verify NATS Migration Prerequisites
 
+From DataMiner 10.6.0 onwards, this Prerequisite is removed as the migration to BrokerGateway has occurred on this version due to being mandatory.<!-- RN 44035 -->
```

Comment:
probably makes more sense as a note, but I guess that's more of a thing to decide by team docs?
In any case, add a whiteline in between to have separate paragraphs